manifests/params.pp

```diff
@@ -27,7 +27,10 @@
       $nodejs_dev_package_ensure = 'absent'
       $nodejs_package_name       = 'nodejs'
       $npm_package_ensure        = 'absent'
-      $npm_package_name          = 'npm'
+      $npm_package_name          = $nodejs::manage_package_repo ? {
+        true    => false,
+        default => 'npm'
+      }
```

Comment:
this produces an interesting circular thingy, which seems to accidentally work on p7 but not on p8

however, using `$manage_package_repo` here is useless, as that is not updated when the users sets `nodejs::manage_package_repo`.

HALP!?